front/app/containers/Admin/projects/project/nativeSurvey/FormResults/FormResultsQuestion/TextQuestion/Analysis/AnalysisInsights.tsx

```diff
@@ -214,31 +235,56 @@ const Question = ({
   );
 };
 
-const AnalysisInsights = ({ analyses }: { analyses: IAnalysisData[] }) => {
-  const { formatMessage } = useIntl();
+const AnalysisInsights = ({ analysis }: { analysis: IAnalysisData }) => {
+  const [automaticSummaryCreated, setAutomaticSummaryCreated] = useState(false);
   const [selectedInsightIndex, setSelectedInsightIndex] = useState(0);
-  const result = useAnalysisInsightsWithIds({
-    analysisIds: analyses?.map((a) => a.id) || [],
+
+  const { formatMessage } = useIntl();
+
+  const { data: inputs } = useInfiniteAnalysisInputs({
+    analysisId: analysis.id,
+    queryParams: {},
```

Comment:
Do we need to include queryParams here even if it's empty?

Reply:
No, you are right. I made them optional.